SimPEG/NSEM/ProblemNSEM.py

```diff
@@ -0,0 +1,560 @@
+from SimPEG.EM.Utils.EMUtils import omega, mu_0
+from SimPEG import SolverLU as SimpegSolver, PropMaps, Utils, mkvc, sp, np
```

Comment:
I would suggest importing scipy.sparse as sp, numpy as np from the base packages - I think down the road we should consider depreciating the import of numpy, scipy from SimPEG